### DISCUSSION TITLE
Github Discussion利用にあたってのプライバシーに関する留意事項 [周知]

### DISCUSSION THREAD

**Discussion by Alpaca-zip** [KBKN-Autonomous-Robotics-Lab/IGVC2023-src#47] [Announcements]
Github Team Discussionsのサポートが切れるということで、これまでのTeam Discussionsの内容をGithub Discussionに移行しました。これに伴い、今のところGithub Discussionの内容は世界中の誰からでも見られるようになってしまっています。本当はIGVC2023-srcをprivateにしたいところですが、ブランチ保護が利用できないなどの面倒が増えるので現状維持にせざるを得ません。**そこで、こちらのTeam Discussionsでは個人が特定されてしまうような内容(名前、住所など)の投稿はお控えいただくようお願いします。**

一通り確認はしましたが、過去の投稿でプライバシーに触れそうな内容があった場合には早めに教えてください。https://github.com/KBKN-Autonomous-Robotics-Lab/IGVC2023-src/discussions/36 で私の名前、所属がバレバレですがこれは別に気にしてないので大丈夫です。

@KBKN-Autonomous-Robotics-Lab/basic-developers

**Comment by Alpaca-zip** [comment]
@koki0624 こちらの件、メールで通知届いていますか？

**Comment by koki0624** [comment]
あれ、メール来てないわ。

**Reply by Alpaca-zip** [reply]
なるほどー。了解。

**Reply by Alpaca-zip** [reply]
Github Discussionを使うときは"@"を使ってチームにメンションする形にしましょう。
